README.md

```diff
@@ -47,3 +47,24 @@ Running this will start your application at [localhost:9292](localhost:9292)
 If you find yourself being rate limited by GitHub - you can define the `GITHUB_TOKEN` environment variable.
 This needs to be a token generated from GitHub, however as the repositories are all public it needs no special
 permissions.
+
+### Security Alerts
```

Comment:
Would mention that all of this happens for free when hitting /security-alerts, just so people don't think they need to do this manually